arches/urls.py

```diff
@@ -283,10 +283,9 @@
     re_path(r"^admin/", admin.site.urls),
     path("i18n/", include("django.conf.urls.i18n")),
     re_path(
-        r"^password_reset/$",
-        PasswordResetView.as_view(),
-        name="password_reset",
-    ),
+    r"^password_reset/$",
+    PasswordResetView.as_view(html_email_template_name="registration/password_reset_html_email.html",extra_email_context={'app_title':settings.APP_TITLE,}),
```

Comment:
I am unsure why there are args being passed into the view here? I'm not seeing them used anywhere in the view itself